packages/@webex/plugin-meetings/src/members/index.ts

```diff
@@ -886,6 +886,31 @@ export default class Members extends StatelessWebexPlugin {
     });
   }
 
+  /**
+   * @param {String} memberId
+   * @param {boolean} [raise] - to raise hand (=true) or lower (=false), default: true
+   * @returns {Promise}
+   * @public
+   * @memberof Members
+   */
+  public moveToLobby(memberId: string) {
+    if (!this.locusUrl) {
+      return Promise.reject(
+        new ParameterError(
+          'The associated locus url for this meetings members object must be defined.'
+        )
+      );
+    }
+    if (!memberId) {
+      return Promise.reject(
+        new ParameterError('The member id must be defined to raise/lower the hand of the member.')
+      );
+    }
+    const options = MembersUtil.getMoveMemberToLobbyRequestOptions(memberId);
```

Comment:
I think there's some naming confusion here. This is named options which calls util ```getMoveMemberToLobbyRequestOptions``` and is passed as the second parameter of ```moveToLobbyMember```. But if you look at ```moveToLobbyMember```, the second parameter is named ```body``` with the first parameter named ```options```.

Suggestion
const body = MembersUtil.getMoveMemberToLobbyRequestBody(memberId);